tests/micro/arduino/test_arduino_rpc_server.py

```diff
@@ -37,8 +37,7 @@
 import conftest
 
 
-# We'll make a new workspace for each test
-@pytest.fixture(scope="function")
+# # A new project and workspace dir is created for EVERY test
```

Comment:
Sorry for the confusion, this should just be:
```suggestion
@pytest.fixture
```